decompy/RL/Action/Action.py

```diff
@@ -0,0 +1,16 @@
+
+class Action:
+    """
+    Step 2: The Driver calls the ActionGenerator to get a list of possible actions for the current
```

Comment:
Actions are used in other places so this description really is incorrect. It would be more useful to have an actual description of what an action represents.